App.js

```diff
@@ -1,50 +1,9 @@
 import React from 'react';
-import { StyleSheet, Text, View, Image} from 'react-native';
+import { StyleSheet, Text, View } from 'react-native';
 import FontLoad from './components/FontLoad';
-import { createBottomTabNavigator, createAppContainer } from 'react-navigation';
-import Notifications from './screens/Notifications';
-import ShopSearch from './screens/ShopSearch';
-import DriverSearch from './screens/DriverSearch';
+import Modal from './components/Modal';
 
-
-const TabNavigator = createBottomTabNavigator({
-  ShopSearch: ShopSearch,
-  Notifications: Notifications,
-  DriverSearch: DriverSearch,
-},
-{
-defaultNavigationOptions: ({ navigation }) => ({
-  tabBarIcon: ({ focused }) => {
-    const { routeName } = navigation.state;
-    let imageName;
-    if (routeName === 'ShopSearch') {
-      imageName = require('./assets/images/nav-cart.png');
-      if (focused) {
-        imageName = require('./assets/images/filled_cart.png');
-      }
-    } else if (routeName === 'Notifications') {
-      imageName = require('./assets/images/home_icon.png');
-      if (focused) {
-        imageName = require('./assets/images/filled_house.png');
-      }
-    } else if (routeName === 'DriverSearch') {
-      imageName = require('./assets/images/person_w_bag.png');
-      if (focused) {
-        imageName = require('./assets/images/filled_person.png');
-      }
-    } 
-    return <Image source={imageName} style={{width: 46, height: 42.99, marginTop:25}} />;
-  },
-}),
-tabBarOptions: {
-  showLabel: false,
-},
-}
-);
-
-export default createAppContainer(TabNavigator);
-
-class App extends React.Component {
+export default class App extends React.Component {
```

Comment:
You have to make sure that you have the same file as in the `master` branch. Call `git pull origin master` to update your branch. I can't merge this until you do!